styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -302,136 +390,3 @@
     color: $brand-primary;
   }
 }
-
-
-//solutions
-
-.solutions-hoverArea {
-  text-decoration: none;
-
-  &:hover {
-
-    .border-white {
-      border-color: $mustard;
-    }
-
-    h6,
-    .rsweb {
-      color: $mustard;
-    }
-  }
-}
-
-
-//quadrant
-
-.quadrant {
-  background-image: url('../imgs/derek/quadrant.jpg');
-  background-position: 23% bottom;
-  background-repeat: no-repeat;
-  background-size: 35%;
-  padding: 20px 0;
-  padding-bottom: 0;
-}
-
-.quadrant-row {
-  @include make-row;
-}
-
-.quadrant-textContainer {
-  @include make-sm-column(6);
-  @include make-sm-column-push(6);
-}
-
-.quadrant-title,
-.quadrant-text {
-  text-align: right;
-}
-
-.quadrant-buttonContainer {
-  @include make-sm-column(6);
-  @include make-sm-column-push(6);
-  @include make-xs-column(6);
-  @include make-xs-column-push(3);
-  padding: $half-padding-full;
-  text-align: right;
-}
-
-.logoShowcase-image {
-  max-height: 40px;
-  max-width: 75%;
-  text-align: center;
-  vertical-align: middle;
-}
```

Comment:
We shouldn't be deleting any CSS or changing existing css.  That doesn't follow the deprecation model we've talked about.  We should create new classes for new elements in the short term and them remove deprecated code at a later time once we're done with it.

Reply:
This code has long been removed from the homepage. It is not in use anywhere on the site and is just adding clutter to the file. No homepage elements will need to be deprecated in order for this to go out. It's simply removing some unused clutter.

Reply:
Either way, code removals usually involve extended QE and it's much easier to confirm from a QE perspective when it's not combined with other changes.